fe1-web/src/features/evoting/network/messages/EndElection.ts

```diff
@@ -17,11 +19,7 @@ export class EndElection implements MessageData {
 
   public readonly registered_votes: Hash;
 
-  constructor(msg: Partial<EndElection>) {
-    if (!msg.election) {
-      throw new ProtocolError("Undefined 'id' parameter encountered during 'EndElection'");
-    }
```

Comment:
😿 ?

Reply:
This check was redundant, it is checked once more here:

https://github.com/dedis/popstellar/blob/work-fe1-nico/fe1-web/src/features/evoting/network/messages/EndElection.ts#L41-L43

Moreover the error message was not accurate and sonarcloud complained about uncovered lines as the code was unreachable due to the double check 😹

Reply:
😸